libs/docs/src/docs/no-version/developing-with-devfiles.md

```diff
@@ -3,78 +3,44 @@ title: Developing with devfiles
 description: Developing with devfiles
 ---
 
-Use the devfile specification to develop a Node.js “Hello World”
-application. Developing this application introduces you to how a devfile
-automates and simplifies your development process.
-
-## Prerequisites
-
-To make developing with the devfile specification easier, consider doing
-the following:
+To get a better understanding of what devfiles can help build, take a look at a few tools that currently support devfile.
 
-- Install [minikube](https://minikube.sigs.k8s.io/docs/start/) to
-  create a Kubernetes cluster on your local machine.
+## Quick start guides
 
-- Download
-  [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-macos/)
-  to access your namespace.
-
-- Enable [ingress on
-  minikube](https://kubernetes.io/docs/tasks/access-application-cluster/ingress-minikube/)
-  to access your devfile project.
+- Use the devfile specification to develop a Node.js “Hello World”
+application. Developing this application introduces you to how a devfile
```

Comment:
I think we should mention the tool in the description just to give it some context.  i.e. "Develop a Node.js "Hello World" application using Odo....."